user_service_basic_test.go

```diff
@@ -0,0 +1,64 @@
+package gimlet
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNaiveUserManager(t *testing.T) {
+	assert := assert.New(t)
+	assert.Implements((*UserManager)(nil), &NaiveUserManager{})
+
+	u, err := NewNaiveUserManager([]*NaiveUser{
+		{
+			User:         "foo",
+			Pass:         "bar",
+			EmailAddress: "baz",
+		},
+	})
+	user, err := u.GetUserByToken(context.Background(), "0:baz:[56 88 246 34 48 172 60 145 95 48 12 102 67 18 198 63]")
```

Comment:
I would make the token string a constant: `expectedToken := fmt.Sprintf("0:baz:%v", md5.Sum([]byte("foo"+"bar")))`.

Reply:
Done.